### DISCUSSION TITLE
Exporting Match Data

### DISCUSSION THREAD

**Discussion by neutronJK** [razaqq/PotatoAlert#50] [General]
Hi,
I want to know if it is possible to export match data to a CSV file so I can bring it into a spreadsheet.  I want to try to figure out how often I am placed on the low WR team by matchmaker and how often the lower WR team actually wins the match.

**Comment by razaqq** [comment]
there already is a setting for that
turn it on and it gets automatically saved as csv
if you know your way around a database the match history is far superior tho
